paddle/fluid/operators/matmul_op.cc

```diff
@@ -318,6 +318,21 @@ class MatMulGradKernel : public framework::OpKernel<T> {
   }
 };
 
+framework::DDim GetDimForInput(const framework::InferShapeContext &ctx,
+                               std::string input_name) {
+  auto shape = ctx.Attrs().Get<std::vector<int>>("shape_" + input_name);
+  auto axis = ctx.Attrs().Get<std::vector<int>>("axis_" + input_name);
+  auto dim = ctx.GetInputDim(input_name);
+  if (!shape.empty() && !axis.empty()) {
+    PADDLE_ENFORCE(
```

Comment:
This one is good, but maybe you need to ask luotao for approval(not sure, let's wait for CI report). In new regulation, || is allowed for PADDLE_ENFORCE should not report error.